doc/ensemble.rst

```diff
@@ -0,0 +1,1189 @@
+.. _ensemble_doc:
+********
+Ensemble
+********
+========
+Overview
+========
+A SmartSim ``Ensemble`` enables users to run a **group** of computational tasks together in an
+``Experiment`` workflow. An ``Ensemble`` is comprised of multiple ``Model`` objects,
+where each ``Ensemble`` member (SmartSim ``Model``) represents an individual application.
+An ``Ensemble`` can be managed as a single entity and
+launched with other :ref:`Models<model_object_doc>` and :ref:`Orchestrators<orch_docs>` to construct AI-enabled workflows.
+
+The :ref:`Ensemble API<ensemble_api>` offers key features, including methods to:
+
+- :ref:`Attach configuration files<attach_files_ensemble>` for use at ``Ensemble`` runtime.
+- :ref:`Load AI models<ai_model_ensemble_doc>` (TF, TF-lite, PT, or ONNX) into the ``Orchestrator`` at ``Ensemble`` runtime.
+- :ref:`Load TorchScripts<TS_ensemble_doc>` into the ``Orchestrator`` at ``Ensemble`` runtime.
+- :ref:`Prevent data collisions<prefix_ensemble>` within the ``Ensemble``, which allows for reuse of application code.
+
+To create a SmartSim ``Ensemble``, use the ``Experiment.create_ensemble()`` API function.
+When creating an ``Ensemble``, it is important to consider one of the **three** ``Ensemble`` creation strategies:
+
+1. :ref:`Parameter expansion<param_expansion_init>`: Generate a variable-sized set of unique simulation instances
+   configured with user-defined input parameters.
+2. :ref:`Replica creation<replicas_init>`: Generate a specified number of copies or instances of a simulation.
+3. :ref:`Manually<append_init>`: Attach pre-configured ``Models`` to an ``Ensemble`` to manage as a single unit.
+
+SmartSim manages ``Ensemble`` instances through the :ref:`Experiment API<experiment_api>` by providing functions to
+launch, monitor, and stop applications.
+
+.. _init_ensemble_strategies:
+==============
+Initialization
+==============
+Overview
+========
+The :ref:`Experiment API<experiment_api>` is responsible for initializing all workflow entities.
+An ``Ensemble`` is created using the ``Experiment.create_ensemble()`` factory method, and users can customize the
+``Ensemble`` creation via the factory method parameters.
+
+The factory method arguments of ``Experiment.create_ensemble()`` are:
+
+-  `name` (str): Specify the name of the ``Ensemble``, aiding in its unique identification.
+-  `params` (dict[str, Any]): Provides a dictionary of {parameters:values} for expanding into the ``Model`` members within the ``Ensemble``. Enables parameter expansion for diverse scenario exploration.
```

Comment:
within the ``Ensemble``. Enables parameter expansion for diverse scenario exploration.

->

within the ``Ensemble``, enabling parameter exploration.